code/game/jobs/job/security.dm

```diff
@@ -304,8 +304,8 @@
 	spawn_positions = 2
 	supervisors = "the Blackshield Commander"
 	difficulty = "Hard."
-	noob_name = "Corpsman Recruit"
-	alt_titles = list("Corpsman Recruit","Combat Medic","Combat Surgeon")
+	noob_name = "Medical Cadet"
```

Comment:
```suggestion
	noob_name = "Corpsman Cadet"
```
I will hold my ground on these being Corpsman Cadet just to keep it uniform sorta-kinda. Since Medical Cadet is kinda more vague than Corpsman is.